test/test-client.js

```diff
@@ -1074,22 +1074,46 @@ describe('Imgix client:', function describeSuite() {
           assert(max <= MAX);
         });
 
-        it('errors with non-Number width', function testSpec() {
+        it('errors with non-Number minWidth', function testSpec() {
           assert.throws(function() {
             new ImgixClient({
               domain: 'testing.imgix.net'
             }).buildSrcSet('image.jpg', {}, {minWidth: 'abc'});
           }, Error);
         });
 
-        it('errors with negative width', function testSpec() {
+        it('errors with negative maxWidth', function testSpec() {
           assert.throws(function() {
             new ImgixClient({
               domain: 'testing.imgix.net'
             }).buildSrcSet('image.jpg', {}, {maxWidth: -100});
           }, Error);
         });
 
+        it('errors with zero maxWidth', function testSpec() {
+          assert.throws(function() {
+            new ImgixClient({
+              domain: 'testing.imgix.net'
+            }).buildSrcSet('image.jpg', {}, {maxWidth: 0});
+          }, Error);
+        });
+
+        it('errors with two invalid widths', function testSpec() {
+          assert.throws(function() {
+            new ImgixClient({
+              domain: 'testing.imgix.net'
+            }).buildSrcSet('image.jpg', {}, {minWidth: 0, maxWidth: 0});
+          }, Error);
+        });
+
```

Comment:
Can we get `it('errors when the maxWidth is less than the minWidth', function testSpec() { ... });`?